scene/3d/skeleton_3d.h

```diff
@@ -175,7 +174,7 @@ class Skeleton3D : public Node3D {
 		NOTIFICATION_UPDATE_SKELETON = 50
 	};
 
-	// skeleton creation api
+	// Skeleton creation api
```

Comment:
// Skeleton creation api.

Added a period. Is this comment necessary?

Reply:
I decided that nouns don't need periods after looking at other sources. Instead, I changed api to API.